### PR TITLE
[PS-1805] BEEEP: Renamed importers based on agreed naming-convention

### PR DESCRIPTION
**Pull request by djsmith85**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
With some plans to change the folder structure of importers, add more tests and possibly split the importers into their own module(actual BEEEP work), I needed to get rid of some tech debt first.

I renamed all importer-related files based on our naming convention which we decided on with https://github.com/bitwarden/adr/blob/master/decisions/0012-angular-filename-convention.md

Removed the entries from the `whitelistlist-capital-letters.txt`

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by differsthecat — APPROVED** [review]
Spent some time looking through this and it looks good to me!
TIL about the whitelist-capital-letters.txt

**Comment by Hinton** [comment]
Could you rename the folders as well while you are at it?

File: libs/common/spec/importers/onepassword-1pux-importer.spec.ts [hunk 1]
```diff
@@ -1,34 +1,34 @@
 import { CipherType } from "@bitwarden/common/enums/cipherType";
 import { FieldType } from "@bitwarden/common/enums/fieldType";
 import { SecureNoteType } from "@bitwarden/common/enums/secureNoteType";
-import { OnePassword1PuxImporter as Importer } from "@bitwarden/common/importers/onepasswordImporters/onepassword1PuxImporter";
+import { OnePassword1PuxImporter as Importer } from "@bitwarden/common/importers/onepasswordImporters/onepassword-1pux-importer";
```
Comment (Hinton) [inline libs/common/spec/importers/onepassword-1pux-importer.spec.ts:4]: This folder name seems to still be wrong?

File: libs/common/spec/importers/test-data/keeper-json/testdata.json.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { KeeperJsonExport } from "@bitwarden/common/importers/keeperImporters/types/keeperJsonTypes";
+import { KeeperJsonExport } from "@bitwarden/common/importers/keeperImporters/types/keeper-json-types";
```
Comment (Hinton) [inline libs/common/spec/importers/test-data/keeper-json/testdata.json.ts:1]: Also wrong?

**Comment by Hinton** [comment]
Please merge in the latest master which has the folder changes to the whitelist, that should let us make certain all folders got renamed. Looks good otherwise.

**Review by Hinton — APPROVED** [review]
Thanks.

**Review by Hinton — APPROVED** [review]

**Pull request merged**